Riot/Modules/Room/RoomViewController.m

```diff
@@ -715,8 +725,8 @@ - (void)viewDidLayoutSubviews
         self.edgesForExtendedLayout = UIRectEdgeLeft | UIRectEdgeBottom | UIRectEdgeRight;
     }
     
-    //  stay at the bottom if already was
-    if (self.isBubblesTableScrollViewAtTheBottom)
+    // re-scroll to the bottom, if at bottom before the most recent layout
+    if (self.shouldScrollToBottomAfterLayout)
```

Comment:
Does it make sense to make `shouldScrollToBottomAfterLayout` false again after we scrolled to bottom?

Reply:
Good point. In practice it would not cause issues without (will/didLayout go in pair), but it will make it more explicit to reset.